### PR TITLE
OCD-2336 -Add the missing calls, explanations and objects for rollout-api

### PR DESCRIPTION
**Pull request by lionelresnik**
Add to openApi(docs) the missing calls, explanations and objects to rollout-api +
small fixes

# Jira Ticket
https://spotinst.atlassian.net/browse/OCD-2336

### CODE REVIEW

**Review by caduri — APPROVED** [review]

File: api/services/ocean/cd/paths/rollout.yaml [hunk 1]
```diff
@@ -1,4 +1,25 @@
 summary: Ocean CD
+get:
+  summary: "Ocean CD Rollout by ID"
+  description: >
+    Ocean CD single rollout.
+  operationId: "OceanCDRolloutSingle"
```
Comment (radchens) [inline api/services/ocean/cd/paths/rollout.yaml:6]: @lionelresnik - Lets change this to DescribeRollout (And every other get)

File: api/services/ocean/cd/paths/rolloutLatest.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+summary: Ocean CD
+get:
+  summary: "Ocean CD Latest Rollouts"
+  description: >
+    Ocean CD Latest rollout/s.
+  operationId: "Ocean CD Latest Rollouts"
```
Comment (radchens) [inline api/services/ocean/cd/paths/rolloutLatest.yaml:6]: @lionelresnik - GetOceanCDLatestRollouts

File: api/services/ocean/cd/paths/rollout.yaml (hunk 1 above)
Comment (lionelresnik) [reply api/services/ocean/cd/paths/rollout.yaml:6]: changed

File: api/services/ocean/cd/paths/rolloutLatest.yaml (hunk 2 above)
Comment (lionelresnik) [reply api/services/ocean/cd/paths/rolloutLatest.yaml]: done

**Review by radchens — APPROVED** [review]

**Pull request merged**